src/v2/Apps/Onboarding/Views/OnboardingDone.tsx

```diff
@@ -0,0 +1,34 @@
+import { FC, useEffect } from "react"
+import { Flex, Spacer, Spinner, Text } from "@artsy/palette"
+import { useOnboardingContext } from "../useOnboardingContext"
+
+export const OnboardingDone: FC = () => {
+  const { next } = useOnboardingContext()
+
+  useEffect(() => {
+    const timeout = setTimeout(next, 5000)
+
+    return () => {
+      clearTimeout(timeout)
+    }
+  }, [next])
+
+  return (
+    <Flex
+      flexDirection="column"
+      alignItems="center"
+      justifyContent="center"
+      textAlign="center"
+    >
+      <Spinner position="static" color="brand" />
+
+      <Spacer mt={2} />
+
+      <Text variant="lg">
+        Great start!
+        <br />
+        We’re adding these to your Artsy home.
+      </Text>
+    </Flex>
```

Comment:
From the designs it looks like we'll have different confirmation steps depending on what screen there was; so there probably won't be a single `Done` step anymore.